cmake/ucm.cmake

```diff
@@ -302,7 +302,8 @@ endmacro()
 macro(ucm_include_file_in_sources)
     cmake_parse_arguments(ARG "" "HEADER" "" ${ARGN})
     if(${ARG_HEADER} STREQUAL "")
-        message(FATAL_ERROR "Need to pass HEADER and a header file to ucm_include_file_in_sources()")
+        
+	(FATAL_ERROR "Need to pass HEADER and a header file to ucm_include_file_in_sources()")

```

Comment:
You've removed the ```message``` call here and the CI builds fail

Reply:
Thanks, that's definitely not a change I wanted to make. Something about how the online GitHub editor jumps around trips me up...